src/Webfactory/Slimdump/Database/Dumper.php

```diff
@@ -60,16 +66,8 @@ public function enableForeignKeys()
     public function dumpSchema($table, Connection $db, $keepAutoIncrement = true, bool $noProgress = false)
     {
         $this->keepalive($db);
-        $this->output->writeln("-- BEGIN STRUCTURE $table", OutputInterface::OUTPUT_RAW);
-        $this->output->writeln("DROP TABLE IF EXISTS `$table`;", OutputInterface::OUTPUT_RAW);
-
-        $tableCreationCommand = $db->fetchColumn("SHOW CREATE TABLE `$table`", [], 1);
-
-        if (!$keepAutoIncrement) {
-            $tableCreationCommand = preg_replace('/ AUTO_INCREMENT=\d*/', '', $tableCreationCommand);
-        }
 
-        $this->output->writeln($tableCreationCommand.";\n", OutputInterface::OUTPUT_RAW);
+        $this->sqlDumper->writeDumpHeadContent($table, $db, $keepAutoIncrement);
```

Comment:
Do you see a way how we could pass the `Dbal\Connection` and the `$keepAutoIncrement` flag to the `SqlDumper`'s constructor?